inc/functions/admin.php

```diff
@@ -8,7 +8,7 @@
  */
 function rocket_need_api_key() {
 	$message = '';
-	$errors  = get_transient( 'rocket_check_key_errors' );
+	$errors  = (array) get_transient( 'rocket_check_key_errors' );
 
 	if ( false !== $errors ) {
```

Comment:
With this change, this condition will never be met, because `$errors` is always an array

Reply:
@Tabrisrp 
I think we don't need this check anymore as we are using foreach,
resolved by removing it completely.